libqtile/backend/wayland/core.py

```diff
@@ -164,6 +165,7 @@ def __init__(self):
         self.add_listener(
             output_power_manager.set_mode_event, self._on_output_power_manager_set_mode
         )
+        self.idle = Idle(self.display)
```

Comment:
```suggestion
        self._idle = Idle(self.display)
```

Reply:
That wouldn't work as it is also called from https://github.com/qtile/qtile/blob/eafc306f12faf6013ecd32e1f6add5a1923f111a/libqtile/backend/wayland/keyboard.py#L118
should i make it into a property?

Reply:
Oh sorry I misread this bit -- your first approach is best